src/Missings.jl

```diff
@@ -207,4 +207,12 @@ missing
 """
 passmissing(f) = PassMissing{Core.Typeof(f)}(f)
 
+
+"""
+    hasmissing(itr)
```

Comment:
May as well be as explicit as possible
```suggestion
    hasmissing(itr) -> Bool
```